merge-v1/archive/mapping-march10-prevalue.ts

```diff
@@ -0,0 +1,202 @@
+//IMPORTANT: THIS MAPPING VERSION WAS THE VERSION THAT WAS DEPLOYED ON MARCH 10TH. DK AND SP REVIEWED THIS VERSION TOGETHER AND DECIDED TO MOVE FORWARD WITH ONLY EXTRACTING THE VALUE FROM SMART CONTRACT DIRECTLY AND WORKING WITHIN THE SUBGRAPH MAPPING FILE TO SORT OUT OTHER FIELD DETAILS FROM THAT. THIS WAY IT WOULD BE FASTER FOR SYNCING.
```

Comment:
Thanks for pointing this out. Not going to make excuses, I have to be more thorough. I completely forgot to run linting and triple-check the .gitignore to include odd files that were added in past commits since the last PR Review.

Reply:
I'll store it in a different location on my local machine, but it is now in my commit history so I can bring it back up the worst case. I think that is what you mean by the last option.